src/features/Classes/ClassDetailPage/index.jsx

```diff
@@ -106,7 +114,21 @@ const ClassDetailPage = () => {
             </Button>
           </div>
           <InstructorCard />
-          <div className="d-flex justify-content-end my-3 flex-wrap">
+          <div className="d-flex justify-content-end my-3 align-items-center ">
+            <Dropdown className="dropdowntpz mx-3">
+              <Dropdown.Toggle
+                as={IconButton}
+                src={MoreVert}
+                iconAs={Icon}
+                variant="primary"
+              />
+              <Dropdown.Menu align="right">
+                <Dropdown.Item onClick={handleGradebookButton}>
+                  <i className="fa-regular fa-book mr-3" />
+                  Gradebook
+                </Dropdown.Item>
+              </Dropdown.Menu>
+            </Dropdown>
```

Comment:
Please left the button first, then this menu and do not forget to update the image in the pr cover letter.